code/controllers/subsystem/SSair.dm

```diff
@@ -16,87 +15,65 @@ SUBSYSTEM_DEF(air)
 	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
 	offline_implications = "Turfs will no longer process atmos, and all atmospheric machines (including cryotubes) will no longer function. Shuttle call recommended."
 	cpu_display = SS_CPUDISPLAY_HIGH
-	var/cost_turfs = 0
-	var/cost_groups = 0
-	var/cost_highpressure = 0
+	var/cost_milla_tick = 0
+	var/cost_interesting_tiles = 0
 	var/cost_hotspots = 0
-	var/cost_superconductivity = 0
 	var/cost_pipenets = 0
 	var/cost_deferred_pipenets = 0
 	var/cost_atmos_machinery = 0
 
-	var/list/excited_groups = list()
-	var/list/active_turfs = list()
+	var/interesting_tile_count = 0
 	var/list/hotspots = list()
 	var/list/deferred_pipenet_rebuilds = list()
 	var/list/networks = list()
 	var/list/atmos_machinery = list()
 	var/list/pipe_init_dirs_cache = list()
 	var/list/machinery_to_construct = list()
 
-
-
-	//Special functions lists
-	var/list/active_super_conductivity = list()
-	var/list/high_pressure_delta = list()
-
 	/// Pipe overlay/underlay icon manager
 	var/datum/pipe_icon_manager/icon_manager
 
-
 	var/list/currentrun = list()
 	var/currentpart = SSAIR_DEFERREDPIPENETS
+	var/has_ticked_milla = FALSE
 
 /datum/controller/subsystem/air/get_stat_details()
 	var/list/msg = list()
 	msg += "C:{"
-	msg += "AT:[round(cost_turfs,1)]|"
-	msg += "EG:[round(cost_groups,1)]|"
-	msg += "HP:[round(cost_highpressure,1)]|"
+	msg += "MT:[round(cost_milla_tick,1)]|"
+	msg += "IT:[round(cost_interesting_tiles,1)]|"
 	msg += "HS:[round(cost_hotspots,1)]|"
-	msg += "SC:[round(cost_superconductivity,1)]|"
 	msg += "PN:[round(cost_pipenets,1)]|"
 	msg += "DPN:[round(cost_deferred_pipenets,1)]|"
 	msg += "AM:[round(cost_atmos_machinery,1)]"
 	msg += "} "
-	msg += "AT:[length(active_turfs)]|"
-	msg += "EG:[length(excited_groups)]|"
+	msg += "IT:[interesting_tile_count]|"
 	msg += "HS:[length(hotspots)]|"
 	msg += "PN:[length(networks)]|"
-	msg += "HP:[length(high_pressure_delta)]|"
-	msg += "AS:[length(active_super_conductivity)]|"
-	msg += "AT/MS:[round((cost ? length(active_turfs)/cost : 0),0.1)]"
 	return msg.Join("")
 
 /datum/controller/subsystem/air/get_metrics()
 	. = ..()
 	var/list/cust = list()
-	cust["active_turfs"] = length(active_turfs)
```

Comment:
So we like katmos now where ATs don't exist and it's just the entire turf grid?

Reply:
- Yup. All turfs are active, always. Even /turf/simulated is a misnomer now: space turfs simulate too, they just set their atmos to 0 every tick.